pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -762,7 +762,8 @@ stdenv.mkDerivation (finalAttrs: {
   strictDeps = true;
 
   nativeBuildInputs = [ removeReferencesTo addDriverRunpath perl pkg-config texinfo yasm ]
-  ++ optionals withCudaLLVM [ clang ];
+  ++ optionals withCudaLLVM [ clang ]
+  ++ (if versionOlder "5" then [ texinfo6 ] else [ texinfo ]);
```

Comment:
```suggestion
  ++ (if versionOlder version "5" then [ texinfo6 ] else [ texinfo ]);
```

Reply:
It seems it's too 🥵 for my 🧠 today, please bear with me, @Atemu.

Reply:
https://lix.systems/ might have some 🍦 for you ;)

Reply:
```sh
% nix --version
nix (Lix, like Nix) 2.90.0-rc1
```

Funnily enough I switch a few days ago, @Atemu 😃